core/src/main/scala/org/http4s/parser/HttpHeaderParser.scala

```diff
@@ -139,9 +139,11 @@ object HttpHeaderParser
     addParser_("RANGE".ci, `RANGE`)
     addParser_("REFERER".ci, `REFERER`)
     addParser_("RETRY-AFTER".ci, `RETRY_AFTER`)
+    addParser_("SEC-WEBSOCKET-VERSION".ci, `SEC_WEBSOCKET_VERSION`)
```

Comment:
I missed the accept header, and I think I also need to add these to a warmup phase